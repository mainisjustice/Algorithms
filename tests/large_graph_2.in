15
0 11 6 7 6 18 5 23 0 15 4 13 13 19 16
11 0 13 23 24 11 2 15 8 8 18 21 12 21 3
6 13 0 3 18 2 1 18 23 13 9 11 1 16 8
7 23 3 0 13 19 24 11 15 20 11 24 5 0 18
6 24 18 13 0 10 21 11 0 6 19 7 2 0 4
18 11 2 19 10 0 15 4 2 3 17 4 12 16 12
5 2 1 24 21 15 0 16 23 18 4 1 7 4 14
23 15 18 11 11 4 16 0 8 24 2 18 3 12 9
0 8 23 15 0 2 23 8 0 19 7 19 10 19 0
15 8 13 20 6 3 18 24 19 0 18 19 21 13 12
4 18 9 11 19 17 4 2 7 18 0 1 5 2 20
13 21 11 24 7 4 1 18 19 19 1 0 17 1 13
13 12 1 5 2 12 7 3 10 21 5 17 0 3 8
19 21 16 0 0 16 4 12 19 13 2 1 3 0 19
16 3 8 18 4 12 14 9 0 12 20 13 8 19 0