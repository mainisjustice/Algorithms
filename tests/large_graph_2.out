0 7 6 7 6 8 5 6 10 11 4 5 7 6 10
7 0 3 6 6 5 2 6 7 8 4 3 4 4 3
6 3 0 3 3 2 1 4 4 5 3 2 1 3 6
7 6 3 0 6 5 4 7 7 8 6 5 4 6 9
6 6 3 6 0 5 4 5 7 6 6 5 2 5 4
8 5 2 5 5 0 3 4 2 3 5 4 3 5 8
5 2 1 4 4 3 0 4 5 6 2 1 2 2 5
6 6 4 7 5 4 4 0 6 7 2 3 3 4 9
10 7 4 7 7 2 5 6 0 5 7 6 5 7 10
11 8 5 8 6 3 6 7 5 0 8 7 6 8 10
4 4 3 6 6 5 2 2 7 8 0 1 4 2 7
5 3 2 5 5 4 1 3 6 7 1 0 3 1 6
7 4 1 4 2 3 2 3 5 6 4 3 0 3 6
6 4 3 6 5 5 2 4 7 8 2 1 3 0 7
10 3 6 9 4 8 5 9 10 10 7 6 6 7 0